rules/type-declaration/tests/Rector/ClassMethod/AddArrayReturnDocTypeRector/Fixture/return_php_doc_should_not_be_replaced_by_less_specific_type_2.php.inc

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace Rector\TypeDeclaration\Tests\Rector\ClassMethod\AddArrayReturnDocTypeRector\Fixture;
+
+final class ReturnPhpDocShouldNotBeReplacedByLessSpecificType
```

Comment:
this as well:

class        : `SkipReturnPhpDoc2`
file name : `skip_return_php_doc2.php.inc`